models/userDao.js

```diff
@@ -2,10 +2,79 @@ import { PrismaClient } from '@prisma/client';
 
 const prisma = new PrismaClient();
 
-// 이메일 불러오기
-const getUserByEmail = async (email) => {};
+const postReview = async (userId, styleCode, color, size, comfort, width) => {
+  const review = await prisma.$queryRaw`
+    INSERT INTO 
+      product_reviews (user_id, style_code, color, size, comfort, width) 
+    VALUES 
+      (${userId}, ${styleCode}, ${color}, ${size}, ${comfort}, ${width});
+  `;
+  return '리뷰가 작성되었습니다.';
+};
 
-// 계정 생성
-const createUser = async () => {};
+const getReview = async (userId, styleCode, color, size, comfort, width) => {
+  const [review] = await prisma.$queryRaw`
+    SELECT 
+      user_id, style_code, color, size, comfort, width
+    FROM 
+      product_reviews
+    WHERE
+      user_id=${userId} and style_code=${styleCode};
+  `;
 
-export default { createUser, getUserByEmail };
+  return review;
+};
+
+const getReviewAverage = async (userId, styleCode) => {
+  const [review] = await prisma.$queryRaw`
+    SELECT 
+      style_code,
+      avg(color) as colorAverage,
+      avg(size) as sizeAverage,
+      avg(comfort) as comfortAverage,
+      avg(width) as widthAverage,
+      (avg(color)+avg(size)+avg(comfort)+avg(width))/4 as totalAverage
+    FROM 
+      product_reviews
+      WHERE
+      style_code=${styleCode};
+    GROUP BY
+      style_code;
+  `;
+
+  return review;
+};
+
+const memberAuthorization = async userId => {
+  const member = await prisma.$queryRaw`
+    UPDATE
+      users
+    SET
+      is_member=1
+    WHERE
+      id=${userId};
+  `;
+
+  return '멤버쉽 가입이 완료되었습니다.';
+};
+
```

Comment:
```suggestion
  return await prisma.$queryRaw`
    UPDATE
      users
    SET
      is_member=1
    WHERE
      id=${userId};
  `;
};

```
사용자에게 전달되는 메세지라면 dao단에서 표현하시는 것은 어색합니다 :)